src/Microsoft.Health.Fhir.Api/Controllers/AadSmartOnFhirProxyController.cs

```diff
@@ -300,10 +337,10 @@ public AadSmartOnFhirProxyController(IOptions<SecurityConfiguration> securityCon
                 decodedCompoundCode = JObject.Parse(Base64UrlEncoder.Decode(compoundCode));
                 code = decodedCompoundCode["code"].ToString();
             }
-            catch
+            catch (Exception ex)
```

Comment:
Is there a particular exception we should be catching here?

Reply:
There are two different exceptions, one from Newtonsoft and another from the Decode, but I think our response is the same for either.  They have no other common base class beside Exception.